libcxx/test/libcxx/feature_test_macro_csv.sh.py

```diff
@@ -0,0 +1,35 @@
+# RUN: %{python} %s %{libcxx-dir}/utils %{libcxx-dir}/utils/data/feature_test_macros/test_data.json
+
+import sys
```

Comment:
Maybe rename to `libcxx/test/libcxx/feature_test_macros/get_table.sh.py`? We'll have other tests in that subdirectory later.